src/main/java/domain/strategy/RandomGenerator.java

```diff
@@ -0,0 +1,12 @@
+package domain.strategy;
+
+import java.util.Random;
+
+public class RandomGenerator implements PointGenerator {
```

Comment:
인터페이스의 이름을 따르는게 좋지 않을까요?

```suggestion
public class RandomPointGenerator implements PointGenerator {
```

Reply:
네! 명확하지 않은 느낌이 있었는데 이젠 확실히 알아볼 수 있게 되었네요! 수정했습니당